compiler/luci/profile/src/CircleNodeOrigin.cpp

```diff
@@ -99,10 +96,8 @@ class CompositeOrigin final : public luci::CircleNodeOrigin
 
     for (auto &origin : _origins)
     {
-      for (auto source : origin->sources())
-      {
-        res.emplace(source);
-      }
+      const auto sources = origin->sources();
+      res.insert(std::begin(sources), std::end(sources));
```

Comment:
why is this changed needed?

Reply:
This is one more approach to refactor the raw loop with a standard library method instead. The `std::set::insert` is able to insert a range of elements by taking 2 iterators.